Dockerfile

```diff
@@ -41,5 +41,5 @@ RUN git add .
 RUN git diff-index --cached --exit-code HEAD
 
 
-RUN go test -v -timeout 11s ./...
+RUN go test -race -v -timeout 60s ./...
```

Comment:
Do we want to have timeouts? Won't the test fail if the race detector finds any data races?

(Anecdote: I tried to set a timeout on Unilog's race detector and the inconsistency convinced me to remove it.)